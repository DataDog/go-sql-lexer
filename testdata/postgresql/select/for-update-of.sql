SELECT * FROM users WHERE last_login < NOW() - INTERVAL '1 year' FOR UPDATE OF users;
