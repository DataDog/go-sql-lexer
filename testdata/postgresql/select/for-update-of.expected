[
    {
        "expected": "SELECT * FROM users WHERE last_login < NOW ( ) - INTERVAL ? FOR UPDATE OF users",
        "statement_metadata": {
            "size": 17,
            "tables": [
                "users"
            ],
            "commands": [
                "SELECT",
                "UPDATE"
            ],
            "comments": [],
            "procedures": []
        }
    }
]